src/bits.rs

```diff
@@ -0,0 +1,23 @@
+use core::{i128, i16, i32, i64, i8, isize};
+use core::{u128, u16, u32, u64, u8, usize};
+
+/// Numbers which have a specified number of bits.
+pub trait Bits {
```

Comment:
What if you called this trait `Precision` instead? Then it could also have `bytes()`